test/acceptance/tests/basic/basic_test.go

```diff
@@ -211,3 +171,26 @@ func TestBasic(t *testing.T) {
 type listTasksResponse struct {
 	TaskARNs []string `json:"taskArns"`
 }
+
+// executeRemoteCommand executes a command inside a container in the task specified
+// by taskARN.
```

Comment:
🥳 

Future improvement: we can have this trim the excess output from the session-manager-plugin:

```
The Session Manager plugin was installed successfully. Use the AWS CLI to start a session.

Starting session with SessionId: ecs-execute-command-0b27d055bac783a77
...

Exiting session with sessionId: ecs-execute-command-0b27d055bac783a77.
```

Then we could parse JSON output from commands easily (instead of resorting to string search or regex).